### DISCUSSION TITLE
这问题。。。

### DISCUSSION THREAD

**Discussion by trustedinster** [Meloong-Git/PCL#3297] [提问] [answered]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/14223023/Log1.txt)

**Comment by MoYuan-CN** [comment, marked as answer]
```
基础连接已经关闭: 未能为 SSL/TLS 安全通道建立信任关系
```

参考 #3178（Edit），也有可能是因为 MCBBS 源关闭的原因？

**Reply by MoYuan-CN** [reply]
最近一堆这个了......

**Reply by allMagicNB** [reply]
还有个问题，LabyMod4

**Reply by hejiehao** [reply]
是 LabyMod 而不是 LabymMod（

**Reply by allMagicNB** [reply]
> 是 LabyMod 而不是 LabymMod（

我知道我打错了所以Edit了（

**Reply by allMagicNB** [reply]
应该是#3178

**Comment by allMagicNB** [comment]
```
[22:33:03.033] [Download] LabyMod-4.jar 511#：出错，文件大小不一致，获取结果为 15263332，要求必须为 15978108。
```
如果你提问的是这个问题，可以参考#3225 ，LabyMod那边好像给的Json文件有问题，开启`版本设置->设置->高级启动选项->关闭文件校验`选项应该就可以了
